products.php

```diff
@@ -0,0 +1,28 @@
+<?php
+header('Content-Type: application/json; charset=utf-8');
+header('Access-Control-Allow-Origin: *');
+
+require('vendor/autoload.php');
+
+use Furniture\Factories\DbConnector;
+use Furniture\Hydrators\ProductHydrator;
+use Furniture\Services\ResponseService;
+use \Furniture\Exceptions\InvalidCategoryException;
+
+const SUCCESS_MESSAGE = "Successfully retrieved products";
+
+try {
+    if (!is_numeric($_GET['cat'])) {
+        throw new InvalidCategoryException('INVALID CATEGORY ID');
+    }
+    $db = DbConnector::getDbConnection();
+    $products = ProductHydrator::fetchProductsByCategoryId($db, $_GET['cat']);
```

Comment:
Since you're doing `$_GET['cat']` twice in this file, better to just assign it to a variable at the start.